src/templates/base/2019/base_ebook.html

```diff
@@ -28,7 +28,7 @@
   @page {
     @top-left { content: ''; }
     @top-right { content: string(chapter-subtitle) string(chapter-title); }
-    @bottom-left { content: "{{ year }} {{ self.organization() }}"; }
+    @bottom-left { content: "{{ year }} {{ self.web_almanac() }} {{ self.preposition() }} {{ self.http_archive() }}"; }
```

Comment:
This doesn't work for Japanese. So think we should maybe keep `self.organization()` (and the associated block).

Reply:
Done.